include/openssl/ssl.h

```diff
@@ -1774,6 +1774,13 @@ OPENSSL_EXPORT STACK_OF(SSL_CIPHER) *SSL_get_ciphers(const SSL *ssl);
 
 // Connection information.
 
+// SSL_in_connect_init returns 1 if |ssl| is a client and has a pending
+// handshake. Otherwise, it returns 0.
+# define SSL_in_connect_init(a)          (SSL_in_init(a) && !SSL_is_server(a))
+// SSL_in_accept_init returns 1 if |ssl| is a server and has a pending
+// handshake. Otherwise, it returns 0.
+# define SSL_in_accept_init(a)           (SSL_in_init(a) && SSL_is_server(a))
```

Comment:
I know OpenSSL defines these as macros, but could we define these as functions here instead? We already do something similar where all the `SSL_CTRL` related functions are redefined as functions in AWS-LC.
https://github.com/aws/aws-lc/blob/92e1332f940e89dcf5e693c1954f436e8237e290/include/openssl/ssl.h#L6028-L6036